request-interfaces.go

```diff
@@ -86,6 +86,15 @@ type LstatFileLister interface {
 	Lstat(*Request) (ListerAt, error)
 }
 
+// RealpathFileLister is a FileLister that implements the Realpath method.
+// We use "/" as start directory for relative paths, implementing this
+// interface you can customize the start directory.
+// You have to return an absolute POSIX path.
+type RealpathFileLister interface {
+	FileLister
+	Realpath(string) string
```

Comment:
In the client we’re using `RealPath`, we might want to keep the two consistent between each other.